autoscheduler/frontend/src/tests/ui/ConfigureCard.test.tsx

```diff
@@ -153,4 +153,47 @@ describe('ConfigureCard component', () => {
       expect(sections.length).toEqual(0);
     });
   });
+
+  describe('shows an error message', () => {
```

Comment:
nit: This should probably be `shows an error snackbar`. Same with the `does not show an error message`.